cloud-service-provider/azure/aks/terraform/opea-chatqna.tfvars

```diff
@@ -0,0 +1,6 @@
+cluster_name = "opea-chatqna"
```

Comment:
let's use cluster name `opea`. The AWS/EKS cluster will change to use that name as well.

Reply:
Updated cluster name to `opea`